src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -258,8 +260,6 @@ private async Task CheckDashboardAvailabilityAsync(string delimitedUrlList, Canc
         if (StringUtils.TryGetUriFromDelimitedString(delimitedUrlList, ";", out var firstDashboardUrl))
         {
             await WaitForHttpSuccessOrThrow(firstDashboardUrl, DashboardAvailabilityTimeoutDuration, cancellationToken).ConfigureAwait(false);
-
-            var distributedApplicationLogger = loggerFactory.CreateLogger<DistributedApplication>();
             distributedApplicationLogger.LogInformation("Now listening on: {DashboardUrl}", firstDashboardUrl.ToString().TrimEnd('/'));
```

Comment:
It also occurs to me that this won't print the dashboard URI if it's not in ASP.NET Core URLS.